src/main/java/com/google/sps/servlets/OverTimeCasesServlet.java

```diff
@@ -75,6 +75,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Gson gson = new Gson();
       String timeReportJson = gson.toJson(toReturn);
       response.getWriter().println(timeReportJson);
+      // Cases in last 7 days for heatmap
+    } else if (lat == 1000.0 && lng == 1000.0) {
```

Comment:
It's better to represent this value as a constant, and give it a meaningful name + add comments: why 1000? what's the magic here?